### PR TITLE
[WIP] Con mongo db

### PR DESCRIPTION
**Pull request by mutabazigakuba**
- added service package
- added the DAO package
- added the model package
-  added the config package
- added the mongodb depencies
- added the spring milestone repo

### CODE REVIEW

File: CrunchifySpringMVCTutorial/src/com/crunchify/controller/CrunchifyHelloWorld.java [hunk 1]
```diff
@@ -6,12 +6,12 @@
 @Controller
 public class CrunchifyHelloWorld {
 	
-	@RequestMapping("/welcome")
+	@RequestMapping("/welcome.jsp")
```
[the remark this answers is not in the document]
Comment (mutabazigakuba) [reply CrunchifySpringMVCTutorial/src/com/crunchify/controller/CrunchifyHelloWorld.java]: I was testing something about having the same naming in the `controller` and `web.xml` and it worked. The latest commit has ` welcome `  not `welcome.jsp`

File: CrunchifySpringMVCTutorial/src/com/crunchify/controller/AuthorController.java [hunk 2]
```diff
@@ -2,11 +2,32 @@
...
+//	@RequestMapping(value="/authors.jsp", method = RequestMethod.GET)
+//	public String passParametersWithModel(Model model) {
+//	    Map<String, String> map = new HashMap<>();
+//	    map.put("spring", "mvc");
+//	    model.addAttribute("message", "hello world");
+//	    model.mergeAttributes(map);
+//	    return "author";
+//	}
+	
+	@RequestMapping(value="/author.jsp", method = RequestMethod.POST)
```
[the remark this answers is not in the document]
Comment (mutabazigakuba) [reply CrunchifySpringMVCTutorial/src/com/crunchify/controller/AuthorController.java]: I did some research and they use params on the requestmapping notation. But it's all the **same** with or without the params  **FOR NOW!**

[the remark this answers is not in the document]
Comment (rrugamba) [reply CrunchifySpringMVCTutorial/src/com/crunchify/controller/AuthorController.java]: keep in mind the difference between `GET` & `POST` requests

[the remark this answers is not in the document]
Comment (rrugamba) [reply CrunchifySpringMVCTutorial/src/com/crunchify/controller/AuthorController.java]: 👍

**Comment by rrugamba** [comment]
Approved pending WIP

**Pull request merged**